RELEASING.md

```diff
@@ -0,0 +1,315 @@
+# Guide to release Apache StormCrawler (Incubating)
+
+## Release Preparation
+
+- Elect a release manager. This can also be done via lazy consensus on the mailing list.
```

Comment:
I think it would be better to write `Select a release manager on the dev mailing list.` Generally we do not want to vote on everything. We should prefer to have multiple volunteers to be release manager.

Reply:
Fixed. Thx!